binary/expression.proto

```diff
@@ -42,18 +42,27 @@ message Expression {
             double fp64 = 11;
             string string = 12;
             bytes binary = 13;
+            
+            // microseconds since epoch
             fixed64 timestamp = 14;
+            
+            // microseconds since epoch (at midnight)
             fixed32 date = 16;
+            
+            // a time value in nanoseconds from midnight.
```

Comment:
Nanos? I thought this was microseconds. In the systems I've worked in the most common representations for time were seconds and microseconds but I think only pandas represents times in nanoseconds. Other systems *can* represents nanos, but that's distinct from choosing to use nanos exclusively as pandas does.

Reply:
Yeah, nanos doesn't make sense. Good point :)